chunky/src/java/se/llbit/chunky/renderer/scene/PreviewRayTracer.java

```diff
@@ -95,6 +95,9 @@ public static boolean nextIntersection(Scene scene, Ray ray) {
     if (scene.isWaterPlaneEnabled()) {
       hit = waterPlaneIntersection(scene, ray) || hit;
     }
+    if (scene.isGroundPlaneEnabled()) {
+      hit |= groundPlaneIntersection(scene, ray);
```

Comment:
This won't call the `groundPlaneIntersection` method if `hit` is already true, eg. if the water plane was hit before. If this gets fixed, we could have ocean grounds for water world mode. :star_struck:

Reply:
I think it should, `|=` uses the bitwise or operator so it can't get short circuited.